bin/host/Cargo.toml

```diff
@@ -16,18 +16,20 @@ workspace = true
 kona-mpt.workspace = true
 kona-client.workspace = true
 kona-common.workspace = true
+kona-derive.workspace = true
 kona-preimage.workspace = true
-kona-derive-alloy.workspace = true
 
 # Alloy
 alloy-rlp.workspace = true
-alloy-eips.workspace = true
-alloy-provider.workspace = true
+alloy-eips = { workspace = true, features = ["kzg"] }
+alloy-serde.workspace = true
+alloy-provider = { workspace = true, features = ["reqwest"] }
```

Comment:
I wonder if these changes are the reason the preimages changed since `alloy-eips` and `alloy-provider` are used outside the blob provider and beacon client